demo/mix.exs

```diff
@@ -57,9 +57,6 @@ defmodule Demo.MixProject do
       {:ex_machina, "~> 2.3"},
       {:hackney, "~> 1.17", override: true},
       {:faker, "~> 0.18"},
-      {:swoosh, "~> 1.0"},
-      {:phoenix_swoosh, "~> 1.0"},
-      {:gen_smtp, "~> 1.1"},
```

Comment:
Are there any other deps we do not use? 🤔

Reply:
Maybe `tesla` and `hackney`?

Reply:
`hackney` is required by Sentry, but I removed `tesla`